harvard-lts/CURIOSity
# Update curiosity container name in README.md

**Update curiosity container name in README.md**
* * *

# What does this Pull Request do?
Updates curiosity container name in README.md to from `harvard-curiosity-web-1 sh` (what it was names in the SoftServ environment) to `curiosity-web-1` 

# How should this be tested?

* Follow the steps in the readme to spin up curiosity from scratch
* The change made is in step 5, when importing the admin login info
* The command`docker exec -it curiosity-web-1 sh` should not give any errors

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl @dl-maura
